templates/analytics/support.html

```diff
@@ -87,6 +87,10 @@ <h3>{{ user.full_name }}</h3>
             <a title="Copy link" class="copy-button" data-copytext="{{ confirmation.url }}">
                 <i class="fa fa-copy"></i>
             </a><br>
+            <b>Link without status change</b>: {{ confirmation.url }}?support=True
```

Comment:
I think calling this `Inspect confirmation page` would be clearer?  @mateuszmandera @hackerkid what do you think?